src/repo.rs

```diff
@@ -1041,6 +1041,32 @@ impl Repository {
         Ok(())
     }
 
+    /// Merge two commits, producing a git_index that reflects the result of
+    /// the merge. The index may be written as-is to the working directory or
+    /// checked out. If the index is to be converted to a tree, the caller
+    /// should resolve any conflicts that arose as part of the merge.
```

Comment:
Can this also include the clauses that start with "The merge performed uses the first common ancestor ..."